Makefile

```diff
@@ -103,6 +107,20 @@ woke: woke-install
 pa11y: pa11y-install html
 	find $(BUILDDIR) -name *.html -print0 | xargs -n 1 -0 $(PA11Y)
 
+vale-get: install
+	. $(VENV); python3 $(SPHINXDIR)/get_vale_conf.py
+	vale > /dev/null 
+
+vale-run: vale-get
+	@echo ""
+	@echo "Running Vale against $(TARGET). To change target set FILE= with make command"
+	@echo ""
+	. $(VENV); vale --config "$(SOURCEDIR)/vale.ini" $(TARGET)
+
+vale-clean:
+	rm -rf ./styles
+	rm -rf vale.ini
```

Comment:
These should have `$(SOURCEDIR)` (or actually `$(SPHINXDIR)` if the files are moved to `.sphinx`).